### ISSUE TITLE
Add Render Settings(graphics settings) 

### ISSUE THREAD

**Issue by JailbreakPapa** [Facepunch/sbox-issues#537]
**How would you like it to work?**
i would like this work as you have a option to scale down object quality like ultra high medium low a render distance setting on how far the players game renders the map

**Comment by SupinePandora43** [comment]
My old GT 640 is dying (100% load) when teying to play sandbox (gamemode).

**Comment by keli5** [comment]
This would be much appreciated, can only get good fps on flatgrass. Every other map starts to put too much load on UHD 630

**Comment by WYVERN2742** [comment]
From a developer's perspective: Would be nice to read this setting so we could disable / change effects and adjust our gamemode accordingly. Not sure how you would implement this on the interface, perhaps as a master quallity slider?

**Comment by SupinePandora43** [comment]
> From a developer's perspective: Would be nice to read this setting so we could disable / change effects and adjust our gamemode accordingly. Not sure how you would implement this on the interface, perhaps as a master quallity slider?

Only as if in dota 2, with an option to use advanced options.

**Comment by SupinePandora43** [comment]
Fps doesn't change between different `SHADER QUALITY` values, quality either.

**Comment by sr229** [comment]
> From a developer's perspective: Would be nice to read this setting so we could disable / change effects and adjust our gamemode accordingly. Not sure how you would implement this on the interface, perhaps as a master quallity slider?

Dota 2 also had custom game content, I think its better to have a global setting all game modes should inherit and respect rather than inventing their own (unless its necessary like additional effects and such like in GMod)
